mkdocs/docs/cli.md

```
@@ -0,0 +1,154 @@
+# CLI
+
+## Old command line interface
+
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)